tests/CMakeLists.txt

```diff
@@ -95,15 +76,11 @@ set(OPENSHOT_TEST_FILES
   Timeline_Tests.cpp)
 
 ##########  SET OPENCV RELATED TEST FILES  ###############
-if(OpenCV_FOUND)
-  set(OPENSHOT_CV_TEST_FILES
+if(ENABLE_OPENCV)
+  list(APPEND OPENSHOT_TEST_FILES
     CVTracker_Tests.cpp
     CVStabilizer_Tests.cpp
     # CVObjectDetection_Tests.cpp
-    )
-  set(OPENSHOT_CV_LIBRARIES
-    ${OpenCV_LIBS}
-    ${PROTOBUF_LIBRARY}
```

Comment:
This incorporates the OpenCV unit tests into the single `openshot-test` executable, the testrunner for all of our unit tests. No need for a separate one.